### ISSUE TITLE
how to load mapviz in rqt

### ISSUE THREAD

**Issue by FrankZhaoSH** [swri-robotics/mapviz#753]
rivz can be loaded into rqt framework, but I haven't yet load mapviz into rqt.

Is it possible to use the mapviz in rqt like using rviz in rqt?

**Comment by pjreed** [comment]
Yes; if you've got mapviz installed, it should appear under Plugins / Visualization / Mapviz:
![image](https://user-images.githubusercontent.com/12546134/150995978-727924d9-ccd8-40aa-bd43-897f3dd7555b.png)

If you're sure you've got it installed but don't see it, try running `rqt --force-discover`, which will force rqt to re-scan its plugin paths.

**Comment by AGG-II** [comment]
Hi, I'm working in Ubuntu 24.04.4 with ROS jazzy on the ros2-devel branch.
I've been trying to run mapviz as a RQT plugin, but I haven't been able to even make RQT show it on the plugins menu.
I've tried it with the apt-get binaries and compiling mapviz on my pc, in both cases I got the same issue. Also, I did run RQT with the "force-discover" option.

**Comment by DangitBen** [comment]
@AGG-II  It seems this was broken at some point. mapviz is missing an entry in its CMakeLists.txt like:
`pluginlib_export_plugin_description_file(rqt_gui "plugin.xml")`

We'll address this in the source shortly, but it _may_ take a while until the next release on the ROS build farm.

**Comment by DangitBen** [comment]
See: https://github.com/swri-robotics/mapviz/pull/875. Feel free to test it out.
